views/page/course/course-preview.html

```diff
@@ -37,21 +37,43 @@ <h5 class="govuk-heading-s">This page is in preview and has not been published y
                     <li class="course-summary-item">
                         <div class="item-container">
                             <span class="item item-header">Course type</span>
-                            <span class="item item-value">{{ course.getType() }}</span>
+                            <span class="item item-value type">{{ course.getType() | default("Unspecified", true) }}</span>
+                        </div>
+                    </li>
+                    <li class="course-summary-item">
+                        <div class="item-container">
+                            <span class="item item-header">Grades</span>
+                            <span class="item item-value">{{ course.getGrades() | default("Unspecified", true) }}</span>
                         </div>
                     </li>
                     <li class="course-summary-item">
                         <div class="item-container">
                             <span class="item item-header">Duration</span>
-                            <span class="item item-value">1 day</span>
+                            <span class="item item-value">{{ course.getDuration() | default("Unspecified", true) }}</span>
                         </div>
                     </li>
                     <li class="course-summary-item">
                         <div class="item-container">
                             <span class="item item-header">Cost</span>
-                            <span class="item item-value">£{{ course.getCost() }} (ex VAT)</span>
+                            <span class="item item-value">£{%- if course.getCost() === 0 -%}
+                                                                Free {%- else -%}
+                                                                {{ course.getCost() }} 
+                                                            {%- endif %}
+                                                            (ex VAT)
+                            </span>
+                        </div>
+                    </li>
+                    <li class="course-summary-item">
+                        <div class="item-container">
+                            <span class="item item-header">Availability</span>
```

Comment:
Can we change this to `Next available date`?

Abi has updated the Jira ticket to reflect this.